### PR TITLE
Media release blog

### PR DESCRIPTION
**Pull request by kendallstrautman**
Could use some editing love from everyone 💗!

### CODE REVIEW

File: content/blog/media-update.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+last_edited: '2020-10-02T15:10:14.859Z'
+---
+
+Media management is a fundamental feature of a CMS. Websites today contain more dog GIFs and landscape hero photos than ever. Content editors need a way to work with that media when creating and updating web pages, blogs, or articles.
+
+Up until now, media changes were handled solely through image fields. By clicking on the field, the editors could upload new images from their local filesystem. There was a 'store' uploading and handling previously added images under the hood, but editors had no way of interacting with that store.
+
+![media-manager-image](/img/media-manager-ui.png)
+
+In this cycle, the Tina Team focused on creating a media manager UI for media stores. The media manager allows editors to upload, delete, and browse media files and directories. File upload can occur via drag and drop directly onto the manager UI or by clicking the 'Upload' button. (add this to the docs)
```
Comment (ncphillips) [inline content/blog/media-update.md:14]: Add a link to the process section of our ROADMAP

```suggestion
In [this cycle](https://github.com/tinacms/tinacms/blob/master/ROADMAP.md#process), the Tina Team focused on creating a media manager UI for media stores. The media manager allows editors to upload, delete, and browse media files and directories. File upload can occur via drag and drop directly onto the manager UI or by clicking the 'Upload' button. (add this to the docs)
```

File: content/blog/media-update.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+3 new attributes were added to the individual [`Media` object interface](/docs/media/#media).
+
+- `type` : Denotes whether the media item is a [file or a directory](https://github.com/tinacms/tinacms/issues/1452).
+- `id`: A unique identifier for this file, typically the full path to the file.
+- `previewSrc`: A URL to source a preview image.
+
+### Media Store
+
+3 new attributes were added to the [Media Store](/docs/media/#media-store) interface.
```
Comment (ncphillips) [inline content/blog/media-update.md:52]:
```suggestion
Two new attributes were added to the [Media Store](/docs/media/#media-store) interface.
```

File: content/blog/media-update.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+```tsx
+/**
+* Style via className------------------------------------
+*/
+<InlineImage
+  name="frontmatter.image.src"
+  uploadDir={() => '/public/images/'}
+  parse={media => media.id}
+  className="inline-img"
+  alt={data.frontmatter.image.alt}
```
Comment (ncphillips) [inline content/blog/media-update.md:99]: Make `className` more obvious? Maybe like...
```suggestion
  alt={data.frontmatter.image.alt}
  className="inline-img" 
```
This is a non-issue really

**Comment by dwalkr** [comment]
Added some revisions.

File: content/blog/media-update.md [hunk 4]
```diff
@@ -129,7 +128,7 @@ The breaking changes are mostly related to both inline and regular image field c
 
 ### _previewSrc_
 
-The `previewSrc` function provides a URL for the image source when the CMS is enabled. This function is implemented by the [media store default](<[https://github.com/tinacms/tinacms/pull/1386](https://github.com/tinacms/tinacms/pull/1386)>) or can be overridden on a field-by-field basis.
+The `previewSrc` function provides a URL for the image source when the CMS is enabled. This function is implemented by the [media store default](\[https://github.com/tinacms/tinacms/pull/1386\](https://github.com/tinacms/tinacms/pull/1386)) or can be overridden on a field-by-field basis.
```
Comment (dwalkr) [inline content/blog/media-update.md:133]: WYSIWYG bug

Comment (dwalkr) [reply content/blog/media-update.md:133]:
```suggestion
The `previewSrc` function provides a URL for the image source when the CMS is enabled. This function is implemented by the [media store default](https://github.com/tinacms/tinacms/pull/1386) or can be overridden on a field-by-field basis.
```

File: content/blog/media-update.md [hunk 5]
```diff
@@ -186,11 +185,11 @@ parse: media => `/images/${media.filename}`
 
 ### _InlineImage_ render child props
 
-When using the [render props pattern](<[https://reactjs.org/docs/render-props.html](https://reactjs.org/docs/render-props.html)>) to configure an inline image field, there was some inconsistency with the props the render child received depending on whether the CMS was enabled. This forced developers to provide a backup source and account for whether props were being passed at all.
+When using the [render props pattern](\[https://reactjs.org/docs/render-props.html\](https://reactjs.org/docs/render-props.html)) to configure an inline image field, there was some inconsistency with the props the render child received depending on whether the CMS was enabled. This forced developers to provide a backup source and account for whether props were being passed at all.
```
Comment (dwalkr) [inline content/blog/media-update.md:190]: ouch

```suggestion
When using the [render props pattern](https://reactjs.org/docs/render-props.html) to configure an inline image field, there was some inconsistency with the props the render child received depending on whether the CMS was enabled. This forced developers to provide a backup source and account for whether props were being passed at all.
```

File: content/blog/media-update.md [hunk 6]
```diff
@@ -186,11 +185,11 @@ parse: media => `/images/${media.filename}`
...
 
 With this new API, the render child is always passed a `src`, and the field handles whether `src` should be the return value from the `previewSrc` function (when the CMS is enabled), or the value in the data source.
 
-> Note that if you're using [gatsby-image](<[https://www.gatsbyjs.com/plugins/gatsby-image/](https://www.gatsbyjs.com/plugins/gatsby-image/)>), you'll still need to provide a backup `src` value as the path to the image is _transformed_ and does not reflect the value in the data source.
+> Note that if you're using [gatsby-image](\[https://www.gatsbyjs.com/plugins/gatsby-image/\](https://www.gatsbyjs.com/plugins/gatsby-image/)), you'll still need to provide a backup `src` value as the path to the image is _transformed_ and does not reflect the value in the data source.
```
Comment (dwalkr) [inline content/blog/media-update.md:194]:
```suggestion
> Note that if you're using [gatsby-image](https://www.gatsbyjs.com/plugins/gatsby-image/), you'll still need to provide a backup `src` value as the path to the image is _transformed_ and does not reflect the value in the data source.
```

**Pull request merged**